pages/bare_metal_cloud/dedicated_servers/move-failover-ip/guide.en-gb.md

```diff
@@ -16,11 +16,15 @@ Additional IP addresses can be moved between the services you use. This provides
 With this technology, you can switch IP addresses from one solution to another in less than a minute, with virtually no interruption to services for your users. It is useful for service migrations (e.g. moving projects from development to production), or when switching to a backup server during a technical issue.
```

Comment:
Do we want to add the word "block" here too or do we leave it like this?

Reply:
@sbraz not really, I am actually going to revert some changes I made and leave the guide as it was.